watertap/costing/tests/test_util_other.py

```diff
@@ -0,0 +1,81 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2023, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+
+import pytest
+import pyomo.environ as pyo
+import idaes.core.util.model_statistics as istat
+
+from idaes.core import (
+    FlowsheetBlock,
+    UnitModelBlock,
+    UnitModelCostingBlock,
+)
+from idaes.core.solvers import get_solver
+from watertap.costing.watertap_costing_package import WaterTAPCosting
+from watertap.costing.util import cost_rectifier, register_costing_parameter_block
+
+
```

Comment:
Maybe you made this separate file for the sake of clarity, but I think this should go in `test_util`